src/ui/title/title_telescope.rs

```diff
@@ -193,15 +193,17 @@ impl TitleTelescope {
     fn draw_loading_chars(&self, text_length: u16, rect: Rect, buf: &mut Buffer) {
         let index_in_text = (self.loading_indicator as u16 % (text_length)) as u16;
         buf.get_mut(rect.x + 1 + index_in_text, rect.y)
-            .set_modifier(Modifier::BOLD);
+            .set_style(Style::default().add_modifier(Modifier::BOLD));
         if index_in_text >= text_length - 2 {
-            buf.get_mut(rect.x + 1, rect.y).set_modifier(Modifier::BOLD);
-            buf.get_mut(rect.x + 2, rect.y).set_modifier(Modifier::BOLD);
+            buf.get_mut(rect.x + 1, rect.y)
+                .set_style(Style::default().add_modifier(Modifier::BOLD));
+            buf.get_mut(rect.x + 2, rect.y)
+                .set_style(Style::default().add_modifier(Modifier::BOLD));
         } else {
             buf.get_mut(rect.x + 1 + index_in_text + 1, rect.y)
-                .set_modifier(Modifier::BOLD);
+                .set_style(Style::default().add_modifier(Modifier::BOLD));
             buf.get_mut(rect.x + 1 + index_in_text + 2, rect.y)
-                .set_modifier(Modifier::BOLD);
+                .set_style(Style::default().add_modifier(Modifier::BOLD));
```

Comment:
aha, the big change in tui 9->10. set_style actually means merge_style in the new version, this caused the enter_folder_small test failure. As I said earlier I filed a bug on it.https://github.com/fdehau/tui-rs/issues/378. Anyway as a TUI coder you need to know the subtle change. So set_style(Style::default) does nothing, whereas it used to mean 'reset to plain'. This code is still not quite right. If the style was , say, italic, it is now italic & bold. in 9 it would be just bold